### DISCUSSION TITLE
URL support for the sync feature of tab sets

### DISCUSSION THREAD

**Discussion by kaycebasques** [pradyunsg/furo#745] [Ideas]
We have a single setup page for macOS, Windows, and Linux. Assume the URL is `https://pigweed.dev/setup.html`. We're using tabs and the tab sync feature to organize the content by OS.

```
---------------------
Install prerequisites
---------------------

.. tab-set::

   .. tab-item:: Linux
      :sync: linux

      ...

   .. tab-item:: macOS
      :sync: macos

      ...

   .. tab-item:: Windows
      :sync: windows

      ...

------------------
Configure settings
------------------

.. tab-set::

   .. tab-item:: Linux
      :sync: linux

      ...

   .. tab-item:: macOS
      :sync: macos

      ...

   .. tab-item:: Windows
      :sync: windows

      ...
```

By default the Linux tabs is shown. We want to be able to provide URLs that load with the Windows tabs shown.

It seems that Furo uses `data-sync-id` HTML attributes to coordinate syncing. We would be OK with something like this: `https://pigweed.dev/setup.html?sync=windows`

**Comment by pradyunsg** [comment]
This isn't really a Furo-specific request, but rather for the extension.

https://github.com/pradyunsg/sphinx-inline-tabs/issues/27
